Cargo.toml

```diff
@@ -1,19 +1,6 @@
-[package]
```

Comment:
Why delete package metadata?

Reply:
I have not found better option to include workspace. That is why I created #12, so we can try to come up with better solution.